app/views/foods/index.html.erb

```diff
@@ -1,10 +1,20 @@
 <h1>Foods</h1>
-<div id="foods">
-  <% @foods.each do |food| %>
-    <%= render food %>
-    <p>
-      <%= link_to "Show this food", food %>
-    </p>
-  <% end %>
+<div class='foods-list'>
+  <table>
+    <tr>
+      <th>Food</th>
+      <th>Measurement Unit</th>
+      <th>Price</th>
+      <th>Action</th>
+    </tr>
+    <% @foods.each do |food| %>
+    <tr>
+      <td><%= food.name%></td>
+      <td><%= food.measurement_unit%></td>
+      <td>$<%= food.price%></td>
+      <td><%= button_to "Delete", food_path(food), method: :delete %></td>
+    </tr>
+    <% end %>
+  </table>
 </div>
-<%= link_to "New food", new_food_path %>
+<%= link_to "New food", new_food_path %>
```

Comment:
Hi @karayamanemre,
You have done a great job in displaying the food list in the view. However, there are a couple of things you can do to improve the code.

- Currently, you are displaying all food items regardless of who the owner is. This should not be the case as per the project requirement. Please make sure to only display food items for the current user. You can do so in the `index` action of the `food controller` like so:
```
def index
   @foods = current_user.foods
end 
```

- The layout of the food list does not match the required layout. Kindly make sure the layouts match. See below a screenshot of your layout and the required layout

### Your layout
![image](https://user-images.githubusercontent.com/20049193/217205453-d269fe74-cf90-474d-9890-a291783431c2.png)

### Required layout
![image](https://user-images.githubusercontent.com/20049193/217205845-5daa39f6-a88e-458e-92ee-a180605ee6f9.png)